src/_posts/2025/2025-03-19-Patch-3818-is-live.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: Patch 3818 is live
+date: 2025-03-19 09:00:00 +0100
+
+categories: community
+type: regular
+status: live
+priority: 700
+
+redirectURL: https://forum.faforever.com/topic/8817/game-version-3818#balance-changes
+image:
+    path: assets/images/2025/Patch-3818.png
+    alt: A graphic rendention of the Seraphim Tier 2 Destroyer, as well as the Seraphim Tier 3 Sniperbot, both of which were major changes in this patch.
+---
+
+The first patch of the year and the latest in a long line is out! Seraphim navy fell under the nerf hammer in hopes of bringing its oppressiveness down, while sniper bots enjoy some well needed up
```

Comment:
The first patch of the year and the latest in a long line is out! -> The first patch of the year and the continued improvements to the balance and game is now live!